src/sestest/sestest.csproj

```diff
@@ -2,7 +2,7 @@
 
   <PropertyGroup>
     <OutputType>Exe</OutputType>
-    <TargetFramework>netcoreapp1.1</TargetFramework>
+    <TargetFramework>netcoreapp2.0</TargetFramework>
     <Company>Microsoft</Company>
     <Authors>Azure Batch</Authors>
     <Product>Azure Batch Software Entitlement Service SDK</Product>
```

Comment:
Consider setting `Authors`, `Company`, `Product`, and `Copyright` in `Directory.Build.props`

Reply:
Good idea. Moved.